spec/integration/_dev/deploy/agent/spec.yml

```diff
@@ -1,5 +1,5 @@
 spec:
-  additionalContents: false
+  additionalContents: true
```

Comment:
Can you please also add a `Dockerfile` into the test package so we can be sure that a package like the one added in https://github.com/elastic/elastic-package/pull/1009 is supported?

Reply:
Added.  Test results below

```
agikthomas@service-integration-dev-idc-01:~/go/src/github.com/elastic/package-spec$ make test | grep 'multi'
# -count=1 is included to invalidate the test cache. This way, if you run "make test" multiple times
=== RUN   TestPatchedSpec/spec_with_multiple_versions_1.0.0
=== RUN   TestPatchedSpec/spec_with_multiple_versions_2.0.0
    --- PASS: TestPatchedSpec/spec_with_multiple_versions_1.0.0 (0.00s)
    --- PASS: TestPatchedSpec/spec_with_multiple_versions_2.0.0 (0.00s)
=== RUN   TestValidateFile/deploy_custom_agent_multi_services
    --- PASS: TestValidateFile/deploy_custom_agent_multi_services (0.04s)
```